src/edu/csus/ecs/pc2/shadow/RemoteEventFeedMonitor.java

```diff
@@ -464,20 +464,19 @@ public void run() {
                                             } catch (Exception e) {
                                                 // TODO design error handling reporting
                                                 if (Utilities.isDebugMode()) {
-                                                    System.err.println("Exception submitting run for: " + event);
+                                                    System.err.println("Exception submitting run for event: " + event + ": " + e);
                                                 }
-                                                log.log(Level.WARNING, "Exception submitting run for: \\" + event);
-                                                
+                                                log.log(Level.WARNING, "Exception submitting run for event: " + event + ": ", e);
                                                 e.printStackTrace();
                                             }
                                         }
 
                                     } catch (Exception e) {
                                         // TODO design error handling reporting (logging?)
                                         if (Utilities.isDebugMode()) {
-                                            System.err.println("Exception processing event: " + e.toString() + ": event = " + event);
+                                            System.err.println("Exception processing event: " + event + "\n " + e.toString());
```

Comment:
Avoid newlines in log messages.   It may seem to read better but it does not allow grep to list both
the message and the event line.   This is inconsistent with nearly all log messages.    Note that even System.err
can be captured to a log file (as it was in the CSUS Contest).

Reply:
Agreed; I'll change it (I had already made such a change in other places; that one slipped by me...)